components/cost-management/policies/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- update_namespace_labels.yaml
+- kyverno_rbac.yaml
```

Comment:
Any chance of a race condition here where Kyverno will attempt to start applying the policy defined in `uodate_namespace_labels.yaml` prior to the RBAC being applied?  I know it appears to be a tiny window.

Reply:
IIRC, kustomize should sort them such that (Cluster)RoleBindings are applied before the ClusterPolicies. Also, if permissions are missing a Kyverno's webhook won't allow ArgoCD to create the Policy. In a few iterations everything should work.